aim/tests/base.py

```diff
@@ -28,10 +28,18 @@
 from aim.db import api
 from aim.db import model_base
 
+
+global_opts = [
```

Comment:
When the APICAPI config options are loaded (for testing the CLI for example) they need apic_system_id to be globally registered otherwise they will break as a consequence of their inheritance